tokens/color/link.json

```diff
@@ -0,0 +1,20 @@
+{
+  "color": {
+    "link": {
+      " ": {"value": "#06c", "type": "color", "themeable": true},
```

Comment:
What does the empty string for key mean here and elsewhere? Is it the default i.e., `$color-link` with no modifiers?